tests/expectations/tests/issue-674-1.rs

```diff
@@ -12,14 +12,11 @@ pub mod root {
         #[allow(unused_imports)]
         use self::super::super::root;
         #[repr(C)]
-        #[derive(Debug, Copy, Clone)]
+        #[derive(Debug, Default, Copy, Clone)]
         pub struct Maybe {
             pub _address: u8,
         }
         pub type Maybe_ValueType<T> = T;
-        impl Default for Maybe {
-            fn default() -> Self { unsafe { ::std::mem::zeroed() } }
-        }
```

Comment:
I think this *should* have been able to derive before; so I think this change is fine, even if I'm not 100% sure what the original issue was.